_sass/custom/custom.scss

```
@@ -309,3 +309,9 @@ button,
   width: 0%;
   z-index: 10000;
 }
+// Fix: Sticky main header (Issue #696)
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)